packages/cli/src/modules/insights/entities/insights-shared.ts

```diff
@@ -31,7 +33,9 @@ export const TypeToNumber = {
 	success: 2,
 	failure: 3,
 } as const;
+// TODO: rename to singular
 export type TypeUnits = keyof typeof TypeToNumber;
+// TODO: rename to singular
 export type TypeUnitNumbers = (typeof TypeToNumber)[TypeUnits];
```

Comment:
Do we need to rename these now (as the comments say)?

Reply:
I renamed, thanks